tests/document_ops.rs

```diff
@@ -217,7 +217,7 @@ fn doc_create_with_policy_grants() -> Result<(), IronOxideErr> {
         },])
     );
 
-    //finally send an empty policy
+    //finally send an empty policy. This will evaluate cleanly and will thus be cached.
```

Comment:
I don't see a test that verifies that we don't cache unclean evaluations?

Reply:
This behavior is exhibited by the integration tests, but it would be easy to add as a unit test as well. I'll do this.